linkerd/docs/namer.md

```diff
@@ -194,9 +194,13 @@ namers:
 ```
 
 Linkerd provides support for service discovery via [Consul](https://www.consul.io/). 
+
 The internal state of the Consul namer can be viewed at the
 admin endpoint: `/namer_state/<prefix>.json`.
 
+The internal store of the Consul dtabs can be viewed at the
```

Comment:
This documentation belongs in namerd/docs/storage.md

Reply:
I'll see to it